cmake/projects/libuv/hunter.cmake

```diff
@@ -7,6 +7,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 include(hunter_report_broken_package)
 
+hunter_add_version(
+  PACKAGE_NAME
+  libuv
+  VERSION
+  1.44.2
+  URL
+  "https://github.com/libuv/libuv/archive/refs/tags/v1.44.2.tar.gz"
```

Comment:
just curiosity: what was patched earlier such that we needed a hunter repo, that is now fixed?

Reply:
yea before was a hunter repro needed but i switched to use direct offical repro. this hunter repros makes no sense if they support offical way of cmake.